docs/reference/cluster/stats.asciidoc

```diff
@@ -505,6 +505,14 @@ Number of fields mapped to the field data type in selected nodes.
 (integer)
 Number of indices containing a mapping of the field data type in selected nodes.
 
+`indexed_vector_count`::
+(integer)
+For dense_vector field types, number of indexed vector types in selected nodes.
+
+`indexed_vector_dims_sum`::
```

Comment:
Now craving some dim sum 😮 

For me it's a little surprising to report a sum of dimensions across fields, would reporting the median be more natural?

Reply:
If there is a way to report number of fields by histogram buckets of dimensions, that'd be great. Otherwise, median and max would make sense.

Reply:
Thanks for your comments, everyone. Just a reminder, that these are stats for a single cluster, where we should expect 1 or several dense vector fields and not much more.  And histogram aggregations can be built during analysis across all clusters of interests. We also don't use `histogram` field types anywhere in our Elasticsearch stats, so I don't know how well they are integrated and supported in monitoring, telemetry, runtime fields and in Kibana's visualizations.

Changing to `median` of dims or `min` and `max` instead of `sum` are good ideas. I will refer to @giladgal which would be more helpful for our analysis.

Reply:
Min and Max makes the most sense to me. I think our main goal is to understand the typical number of dimensions and I think median could be misleading in that sense because it may hide the extremes. In practice I think max would be the more important one because the main goal is to understand the scalability and latency needs that sometimes relate to the number of dimensions.